Tests/scripts/collect_tests/collect_tests.py

```diff
@@ -885,6 +885,11 @@ def _collect_yml(self, content_item_path: Path) -> CollectionResult | None:
 
             case FileType.SCRIPT | FileType.PLAYBOOK:
                 try:
+
+                    if "ApiModule" in yml.id_:
+                        logger.debug(f"Found changes in ApiModule = {yml.id_}")
```

Comment:
add to the log, what are the implications? What happens because we found changes to the ApiModule?